net/netxng/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ ROOT_STANDARD_LIBRARY_PACKAGE(NetxNG
     src/TNetXNGSystem.cxx
     src/RRawFileNetXNG.cxx
   LIBRARIES
-    Xrootd::Xrootd
+    ${XROOTD_LIBRARIES}
```

Comment:
We should try to link only against the **client** libraries. I bet there's a part in the upstream `FindXROOTD.cmake` that allows for this?

Reply:
I think there is. I'll try to use it

Reply:
It works, the PR is updated